jib-core/src/main/java/com/google/cloud/tools/jib/api/DockerDaemonImage.java

```diff
@@ -70,7 +71,7 @@ private DockerDaemonImage(ImageReference imageReference) {
    * @param dockerExecutable the path to the {@code docker} CLI
    * @return this
    */
-  public DockerDaemonImage setDockerExecutable(Path dockerExecutable) {
+  public DockerDaemonImage setDockerExecutable(@Nullable Path dockerExecutable) {
```

Comment:
I think javadoc should say something about `null` meaning default, though I lean towards not having `null` mean default and having the caller do the null check.